cmd/geth/dbcmd.go

```diff
@@ -684,3 +701,50 @@ func exportChaindata(ctx *cli.Context) error {
 	db := utils.MakeChainDatabase(ctx, stack, true)
 	return utils.ExportChaindata(ctx.Args().Get(1), kind, exporter(db), stop)
 }
+
+func showMetaData(ctx *cli.Context) error {
+	stack, _ := makeConfigNode(ctx)
+	defer stack.Close()
+	db := utils.MakeChainDatabase(ctx, stack, true)
+	ancients, err := db.Ancients()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error accessing ancients: %v", err)
```

Comment:
You probably also want to `return` here

Reply:
Actually, that was intentional. This command should give the caller as much data as possible, so I don't want to error out just because one detail is wrong. As it is, we spit out an error and proceed to give as much information as we can.